docs/runtimes/databricks.ipynb

```diff
@@ -4,7 +4,6 @@
    "cell_type": "markdown",
```

Comment:
<p>You have the same logic above I think it would be better to reuse this for </p><h2>Example of running a Databricks job from a local file</h2><p>as well</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/4979/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>